src/app/components/video/video.component.html

```diff
@@ -0,0 +1,6 @@
+<video autoplay loop>
+    <source
+            src="../../assets/mp4"
+            type="video/mp4">
+            Your browser does not support this version.
+</video>
```

Comment:
this file needs to be prettiered

Reply:
https://www.w3schools.com/tags/tag_video.asp yup.

Reply:
oh lol
the tag I was talking about didn't render in the markdown bc markdown tries to read html tags. I meant `</source>`

Reply:
but interesting, what's the difference between using a source html element and using the src attribute on the video element?

Reply:
you can specify type.